File: clients/java/android/src/main/java/com/wordnik/client/model/TradeBin.java
package com.wordnik.client.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.Date;
public class TradeBin {
  @JsonProperty("timestamp")
  private Date timestamp = null;
  @JsonProperty("symbol")
  private String symbol = null;
  @JsonProperty("open")
  private Double open = null;
  @JsonProperty("high")
  private Double high = null;
  @JsonProperty("low")
  private Double low = null;
  @JsonProperty("close")
  private Double close = null;
  @JsonProperty("trades")
  private Double trades = null;
  @JsonProperty("volume")
  private Double volume = null;
  @JsonProperty("vwap")
  private Double vwap = null;
  @JsonProperty("lastSize")
  private Double lastSize = null;
  @JsonProperty("id")
  private Double id = null;
  public Date getTimestamp() {
    return timestamp;
  }
  public void setTimestamp(Date timestamp) {
    this.timestamp = timestamp;
  }

  public String getSymbol() {
    return symbol;
  }
  public void setSymbol(String symbol) {
    this.symbol = symbol;
  }

  public Double getOpen() {
    return open;
  }
  public void setOpen(Double open) {
    this.open = open;
  }

  public Double getHigh() {
    return high;
  }
  public void setHigh(Double high) {
    this.high = high;
  }

  public Double getLow() {
    return low;
  }
  public void setLow(Double low) {
    this.low = low;
  }

  public Double getClose() {
    return close;
  }
  public void setClose(Double close) {
    this.close = close;
  }

  public Double getTrades() {
    return trades;
  }
  public void setTrades(Double trades) {
    this.trades = trades;
  }

  public Double getVolume() {
    return volume;
  }
  public void setVolume(Double volume) {
    this.volume = volume;
  }

  public Double getVwap() {
    return vwap;
  }
  public void setVwap(Double vwap) {
    this.vwap = vwap;
  }

  public Double getLastSize() {
    return lastSize;
  }
  public void setLastSize(Double lastSize) {
    this.lastSize = lastSize;
  }

  public Double getId() {
    return id;
  }
  public void setId(Double id) {
    this.id = id;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class TradeBin {\n");
    sb.append("  timestamp: ").append(timestamp).append("\n");
    sb.append("  symbol: ").append(symbol).append("\n");
    sb.append("  open: ").append(open).append("\n");
    sb.append("  high: ").append(high).append("\n");
    sb.append("  low: ").append(low).append("\n");
    sb.append("  close: ").append(close).append("\n");
    sb.append("  trades: ").append(trades).append("\n");
    sb.append("  volume: ").append(volume).append("\n");
    sb.append("  vwap: ").append(vwap).append("\n");
    sb.append("  lastSize: ").append(lastSize).append("\n");
    sb.append("  id: ").append(id).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: clients/csharp/src/Swagger/Client/Model/TradeBin.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Swagger.Client.Model {
  public class TradeBin {
    public DateTime? timestamp { get; set; }

    public string symbol { get; set; }

    public double? open { get; set; }

    public double? high { get; set; }

    public double? low { get; set; }

    public double? close { get; set; }

    public double? trades { get; set; }

    public double? volume { get; set; }

    public double? vwap { get; set; }

    public double? lastSize { get; set; }

    public double? id { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TradeBin {\n");
      sb.Append("  timestamp: ").Append(timestamp).Append("\n");
      sb.Append("  symbol: ").Append(symbol).Append("\n");
      sb.Append("  open: ").Append(open).Append("\n");
      sb.Append("  high: ").Append(high).Append("\n");
      sb.Append("  low: ").Append(low).Append("\n");
      sb.Append("  close: ").Append(close).Append("\n");
      sb.Append("  trades: ").Append(trades).Append("\n");
      sb.Append("  volume: ").Append(volume).Append("\n");
      sb.Append("  vwap: ").Append(vwap).Append("\n");
      sb.Append("  lastSize: ").Append(lastSize).Append("\n");
      sb.Append("  id: ").Append(id).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
